packages/jest-types/src/Global.ts

```diff
@@ -134,6 +134,6 @@ export interface GlobalAdditions extends TestFrameworkGlobals {
 export interface Global
   extends GlobalAdditions,
     // TODO: Maybe add `| Window` in the future?
```

Comment:
I believe this is handled by `globalThis`

```suggestion
```

Reply:
done